BaseTools/Source/C/Include/IndustryStandard/PeImage.h

```diff
@@ -666,7 +666,7 @@ typedef struct {
   UINT32  FunctionStartAddress;
   UINT32  FunctionEndAddress;
   UINT32  UnwindInfoAddress;
-} RUNTIME_FUNCTION;
+} PD_RUNTIME_FUNCTION; // MU_CHANGE: Resolve definition conflict for ARM

```

Comment:
Can we redefine this industry standard definition? i.e. is it spec defined? Sure it's just a name, but still...perhaps using `!if x86` around this block and the relevant code blocks above is worthwhile, if terribly annoying.

Reply:
What does it conflict with on ARM builds?

Reply:
`RUNTIME_FUNCTION` from <winnt.h> in Windows Kit, which was pulled in when including <windows.h>. This definition is somehow only left out when building for x86...

Reply:
Okay. I think changing simply changing the type here is a good approach. It is simple, only affects the BaseTools instance of the header (which would potentially conflict with tool chain types and already contains a lot of differences to the non-BaseTools version), and clearly differentiates itself from the winnt.h type.

I'd prefer avoiding an architecture conditional to simplify type usage.

Updating the comment to specifically mention winnt.h would be helpful to point people to the exact source of the conflict.

Reply:
Thanks, this is updated in the comment for structure definition.

Reply:
Will close this conversation for now as no more comments with the current approach. I will follow up if there is any feedback from edk2 community when i send them the patches.